<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>功能区设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ribbonframe.qrc">
    <normaloff>:/icon/res/settings.png</normaloff>:/icon/res/settings.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>382</width>
        <height>308</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>功能区设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="enableHideRibbonCheck">
         <property name="text">
          <string>允许隐藏功能区</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ribbonPinCheck">
         <property name="text">
          <string>总是显示功能区</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="doubleClickShowHideRibbonCheck">
         <property name="text">
          <string>双击Ribbon标签显示/隐藏功能区</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>功能区隐藏时，点击Ribbon标签：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="showWhenTabClickedComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="showLeftNaviBarCheck">
         <property name="text">
          <string>显示左侧导航栏（需要重新启动）</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showStatusbarCheck">
         <property name="text">
          <string>显示状态栏</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>主题设置</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>选择主题：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="selectStyleCombo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="followSystemColorModeCheck">
         <property name="text">
          <string>跟随系统深色/浅色主题</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>主题颜色：</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="themeColorHLayout">
         <item>
          <widget class="QGroupBox" name="colorPresetGroupbox">
           <property name="title">
            <string>预置颜色</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5"/>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCustomColor">
           <property name="text">
            <string>自定义主题颜色...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="followSystemThemeColorCheck">
         <property name="text">
          <string>跟随系统主题颜色</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ribbonframe.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
